configurations/flyway/src/main/java/io/micronaut/configuration/dbmigration/flyway/management/endpoint/FlywayMigration.java

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright 2017-2018 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.micronaut.configuration.dbmigration.flyway.management.endpoint;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import org.flywaydb.core.api.MigrationInfo;
+import org.flywaydb.core.api.MigrationState;
+import org.flywaydb.core.api.MigrationType;
+
+import java.time.Instant;
+import java.util.Date;
+
+/**
+ * A Flyway migration.
+ *
+ * @author Iván López
+ * @see MigrationInfo
+ * @since 1.1
+ */
+@JsonInclude(JsonInclude.Include.ALWAYS)
+public class FlywayMigration {
```

Comment:
What value is this class providing? Why not simply render the `MigrationInfo` as json?